runtime/src/bank.rs

```diff
@@ -4622,9 +4623,15 @@ impl Bank {
             };
 
             let mut compute_budget = ComputeBudget::default();
-            let prioritization_fee = compute_budget
-                .process_message(message, false, false, prioritization_fee_type_change)
+            let prioritization_fee_details = compute_budget
+                .process_instructions(
+                    message.program_instructions_iter(),
+                    false,
+                    false,
```

Comment:
@jstarry @taozhu-chicago We are specifying the default unit feature as false here when calculating prioritization fee.  Was this intentional?  Doing so makes it hard to cleanup the default units per instruction feature because with the above `process_instruction` has to support both.  Any reason we can't use default units per ix when calculating prioritization fee?

Reply:
Looks like value `false` was inherited from this [commit](https://github.com/solana-labs/solana/commit/e070c5ca385284efc45bee10e63ce10550b078fd) when this parameter was initially introduced. 

I agree it doesn't have to be `false`, as matter of fact, priority related code assume `true` for  it to use default unit features

Reply:
Just to clarify, I think PR #26684 is valid, cross the cluster should use "default units per instruction", therefore feature `J2Qd` can be cleared out.